install.zsh

```diff
@@ -20,14 +46,10 @@ main() {
 
     git clone -b "${BRANCH:-master}" https://github.com/zap-zsh/zap.git "$ZAP_DIR" > /dev/null 2>&1 || { echo "❌ Failed to install Zap" && return 2 }
 
-# @formatter:off
-	if ! grep -q '[ -f "$HOME/.local/share/zap/zap.zsh" ] && source "$HOME/.local/share/zap/zap.zsh"' "$ZSHRC"; then
```

Comment:
Did you not want to keep this check in the new version or some other method of identifying that Zap was previously installed?  For those times a user trys to run the install a second time.

Reply:
Right I'll add this back